runtime/doc/builtin.txt

```diff
@@ -4261,6 +4263,36 @@ getreginfo([{regname}])					*getreginfo()*
 		Can also be used as a |method|: >
 			GetRegname()->getreginfo()
 
+getregion({pos1}, {pos2}, {type})			*getregion()*
+		Returns the list of strings from {pos1} to {pos2}.
+		For possible values of {pos1} and {pos2} see |line()|.
+		The {type} is region type:
```

Comment:
The third argument should be an optional Dict.  This will allow us to add additional optional arguments as Dict items in the future.  Otherwise, you need to specify all the mandatory arguments.

Reply:
I don't think `type` should be optional.

The optional arguments should be added later.

Reply:
Agree, the selection type should always be given, no?